src/Tribe/Models/Post_Types/Venue.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Models an Venue.
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Events\Models\Post_Types
+ */
+namespace Tribe\Events\Models\Post_Types;
+
+use Tribe\Models\Post_Types\Base;
+use Tribe\Utils\Lazy_Collection;
```

Comment:
Line 12 to 16 are unused imports; if those are still not used, then clean the imports.